From owner-linux-pci@atrey.karlin.mff.cuni.cz Thu Mar 22 03:53:50 2007
From: Michael Ellerman <michael@ellerman.id.au>
To: linux-pci@atrey.karlin.mff.cuni.cz
Cc: Greg Kroah-Hartman <greg@kroah.com>,
	Eric W.Biederman <ebiederm@xmission.com>,
	David S.Miller <davem@davemloft.net>,
	Benjamin Herrenschmidt <benh@kernel.crashing.org>,
	Andrew Morton <akpm@osdl.org>,
	<daniel.e.wolstenholme@intel.com>
Date: Thu, 22 Mar 2007 21:51:34 +1100
Subject: MSI: Move EXPORT_SYMBOL()s near their definition
Message-Id: <20070322105337.B48D5DDF63@ozlabs.org>

Move EXPORT_SYMBOL()s near their definition.

Signed-off-by: Michael Ellerman <michael@ellerman.id.au>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/pci/msi.c |    9 ++++-----
 1 file changed, 4 insertions(+), 5 deletions(-)

--- a/drivers/pci/msi.c
+++ b/drivers/pci/msi.c
@@ -550,6 +550,7 @@ int pci_enable_msi(struct pci_dev* dev)
 	status = msi_capability_init(dev);
 	return status;
 }
+EXPORT_SYMBOL(pci_enable_msi);
 
 void pci_disable_msi(struct pci_dev* dev)
 {
@@ -581,6 +582,7 @@ void pci_disable_msi(struct pci_dev* dev
 
 	dev->first_msi_irq = 0;
 }
+EXPORT_SYMBOL(pci_disable_msi);
 
 static int msi_free_irq(struct pci_dev* dev, int irq)
 {
@@ -673,6 +675,7 @@ int pci_enable_msix(struct pci_dev* dev,
 	status = msix_capability_init(dev, entries, nvec);
 	return status;
 }
+EXPORT_SYMBOL(pci_enable_msix);
 
 static void msix_free_all_irqs(struct pci_dev *dev)
 {
@@ -706,6 +709,7 @@ void pci_disable_msix(struct pci_dev* de
 
 	msix_free_all_irqs(dev);
 }
+EXPORT_SYMBOL(pci_disable_msix);
 
 /**
  * msi_remove_pci_irq_vectors - reclaim MSI(X) irqs to unused state
@@ -732,8 +736,3 @@ void pci_no_msi(void)
 {
 	pci_msi_enable = 0;
 }
-
-EXPORT_SYMBOL(pci_enable_msi);
-EXPORT_SYMBOL(pci_disable_msi);
-EXPORT_SYMBOL(pci_enable_msix);
-EXPORT_SYMBOL(pci_disable_msix);
